test/unit/test_cons.c

```diff
@@ -324,13 +324,48 @@ bool test_line_kill_word(void) {
 	mu_end;
 }
 
+bool test_line_undo(void) {
+	RzCons *cons = rz_cons_new();
+	// Make test reproducible everywhere
+	cons->force_columns = 80;
+	cons->force_rows = 23;
+	rz_line_free();
+	RzLine *line = rz_line_new();
+	line->ns_completion.run = onecompletion_run;
+	cons->line = line;
```

Comment:
I think this is not necessary. Did you just copy this from `test_line_onecompletion` or was it actually required for the test to work?